tests/test_Deflectors/test_richness2mass.py

```diff
@@ -0,0 +1,9 @@
+from slsim.Deflectors.richness2mass import mass_richness_relation
+
+
+def test_mass_richness_relation():
+    mass = mass_richness_relation(40, relation="simet2017")
+    assert mass > 0
+
+    mass = mass_richness_relation(40, relation="Abdullah2022")
+    assert mass > 0
```

Comment:
Here, you can compute mass using that relation and compare this number with this mass here.Then, If someone later comes and changes this relation, this test will fail.

Reply:
updated the test to an specific one